crates/xline/src/utils/trace.rs

```diff
@@ -65,3 +90,56 @@ pub fn init_subscriber(
         .try_init()?;
     Ok(guard)
 }
+
+/// initialize the subscriber to terminal
+#[inline]
+fn init_subscriber_std(
+    name: &str,
+    log_config: &LogConfig,
+    trace_config: &TraceConfig,
+) -> Result<Option<WorkerGuard>> {
+    let stdout_layer = tracing_subscriber::fmt::layer()
+        .event_format(format().compact())
+        .with_writer(std::io::stderr)
+        .with_ansi(false)
+        .with_filter(*log_config.level());
+
+    let jaeger_level = *trace_config.jaeger_level();
+    let tmp = trace_config
+        .jaeger_online()
+        .then(|| {
+            opentelemetry_jaeger::new_agent_pipeline()
```

Comment:
The `opentelemetry_jaeger::new_agent_pipeline` is deprecated since 0.21.0. We'd better migrate to `opentelemetry-otlp` exporter.  FYI: https://docs.rs/opentelemetry-jaeger/latest/opentelemetry_jaeger/fn.new_agent_pipeline.html